File: todos-java/src/main/java/de/welcz/todosjava/Todo.java
package de.welcz.todosjava;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.*;
import org.bson.types.ObjectId;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "todos")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Todo {
  @Id
  @JsonIgnore
  @With(AccessLevel.PUBLIC)
  private ObjectId id;
  private String title;
}

File: todos-coroutini/src/main/kotlin/de/welcz/todoscoroutini/Todo.kt
package de.welcz.todoscoroutini

import com.fasterxml.jackson.annotation.JsonIgnore
import org.bson.types.ObjectId
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "todos")
data class Todo(
    @Id
    @JsonIgnore
    val id: ObjectId? = null,
    val title: String,
)
